bokehjs/src/coffee/models/mappers/log_mapper.coffee

```diff
@@ -20,11 +21,14 @@ export class LogMapper extends Model
     if inter_scale == 0
       value = 0
     else
-      value = (Math.log(x) - inter_offset) / inter_scale
-      if isNaN(value) or not isFinite(value)
-        value = 0
+      clamped_x = clamp(x, @source_range.start, @source_range.end)
+      _x = (Math.log(clamped_x) - inter_offset) / inter_scale
+      if isFinite(_x)
+        value = _x * scale + offset
+      else
```

Comment:
I think so, there's not much else we can do with `inf` at the low level besides "not draw"